Bottom: 327bd5efcfa4314dba3fd2f3726d4098c591ea73
Top:    e82ce5e2fed3aa678bd1b9a292c06d7cb849a4f3
Author: Jon Smirl <jonsmirl@terra>
Date:   2012-06-29 12:31:28 -0400

From b485de4ab2a5e7ff22f4e7b5a19c470a2c9f4db7 Mon Sep 17 00:00:00 2001
Subject: Make sure mmc stop command is fully initialized before issuing the
 parent command


---

diff --git a/drivers/mmc/host/lpc31xx_mmc.c b/drivers/mmc/host/lpc31xx_mmc.c
index f04d4c7..31b8f78 100644
--- a/drivers/mmc/host/lpc31xx_mmc.c
+++ b/drivers/mmc/host/lpc31xx_mmc.c
@@ -611,11 +611,10 @@ static void lpc313x_mci_start_request(struct lpc313x_mci *host,
 	if (data) //we may need to move this code to mci_start_command
 		lpc313x_mci_submit_data(host, data);
 
-	lpc313x_mci_start_command(host, cmd, cmdflags);
-
 	if (mrq->stop) 
 		host->stop_cmdr = lpc313x_mci_prepare_command(slot->mmc, mrq->stop);
-	
+
+	lpc313x_mci_start_command(host, cmd, cmdflags);
 }
